### DISCUSSION TITLE
Where can I find the logs to check for errors when exiting labwc?

### DISCUSSION THREAD

**Discussion by charbelnicolas** [labwc/labwc#1410] [Q&A] [answered]
Every time I exit labwc I get a couple of warnings and errors on the screen but they get cleared too fast for me to read. I've managed to take a video but I would like to know if there is a place where this is being logged?

I aready tried journalctl -b 1 but didn't find anything related to what I'm seeing when exiting labwc.

Any help is appreciated, thank you.

**Comment by johanmalm** [comment, marked as answer]
labwc 2> log.txt

**Reply by charbelnicolas** [reply]
Thank you!

**Comment by charbelnicolas** [comment]
Ok, I got the log now! Any idea if these can be fixed? Is this something that has to do with labwc tablet support?

```
00:00:00.151 [ERROR] [libseat] [libseat/backend/logind.c:137] Could not take device: No such device
00:00:00.151 [ERROR] [backend/session/session.c:321] Failed to open device: '/sys/devices/pci0000:00/0000:00:14.0/usb1/1-11/1-11:1.0/0003:056A:0357.0007/input/input21/input21::wacom-0.3/brightness': Resource temporarily unavailable
libinput error: event16 - Wacom Intuos Pro M Pad: unable to init LED group: No such file or directory
```

And would you happen to know how to get rid of this error?

`[005 21:58:16.551168] [glfw error 65544]: process_desktop_settings: failed with error: [org.freedesktop.DBus.Error.UnknownMethod] No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop`

I already have xdg-desktop-portal, xdg-desktop-portal-wlr and xdg-desktop-portal-gtk installed and running.

Thank you! I appreciate your help.

**Comment by jp7677** [comment]
> And would you happen to know how to get rid of this error?

This looks more like your OS setup (Wacom firmware/udev/kernel?). The question is, do you experience any hinder or does the tablet just works?

**Reply by charbelnicolas** [reply]
The tablet works fine, just wondering if it is possible to get rid of the error.

**Reply by charbelnicolas** [reply]
I don't get these wacom tablet errors under x11 though!
